src/PhpPact/Standalone/Runner/ProcessRunner.php

```diff
@@ -17,72 +17,48 @@
  */
 class ProcessRunner
 {
-    /** @var Process */
-    private $process;
+    private Process $process;
 
-    /** @var string command output */
-    private $output;
+    private string $output = '';
 
     /** @var int command exit code */
-    private $exitCode;
+    private int $exitCode;
 
-    /** @var string */
-    private $stderr;
+    private string $stderr = '';
 
-    /**
-     * @var LoggerInterface
-     */
-    private $logger;
+    private ?LoggerInterface $logger = null;
 
     /**
-     * @param string $command
-     * @param array  $arguments
+     * @param array<int, string> $arguments
      */
     public function __construct(string $command, array $arguments)
     {
         $this->exitCode  = -1;
         $this->process   = new Process($command . ' ' . \implode(' ', $arguments));
     }
 
-    /**
-     * @param LoggerInterface $logger
-     *
-     * @return ProcessRunner
-     */
     public function setLogger(LoggerInterface $logger): self
     {
         $this->logger = $logger;
 
         return $this;
     }
 
-    /**
-     * @return string
-     */
-    public function getOutput(): ?string
+    public function getOutput(): string
```

Comment:
If you agree with my suggestion above, this code can be reverted:

```suggestion
    public function getOutput(): ?string
```

Reply:
@tienvx There are too much cases, where code expect string, with nullable result we should add a bit more checks with no benefits. It's reasonable?

I think we should commit it as is and perhaps improve in future pull requests.

Reply:
@tienvx resolve your comment, if agree

Reply:
I agree. Please resolve those comments related to `output` and `stderror`. I don't see any button to resolve them. I think only author of the PR can.